docs/tutorials/upgrading-ev3dev-using-shell

```diff
@@ -0,0 +1,24 @@
+---
+title: Upgrading ev3dev Using Shell
+subject: Upgrading
+---
+
+Once you've written the ev3dev image to the SD card, booted, and setup a network 
+connection, you may want to update to the latest kernel.  To do so:
+
+1. Connect to the EV3 using SSH.  (Wireless or USB.)  The default login is 
```

Comment:
ev3dev supports hardware other than EV3s (e.g. BrickPi, PiStorms, EVB) so maybe call it "the ev3dev device."